index.js

```diff
@@ -152,6 +162,24 @@ const create = (win, options) => {
 					});
 				}
 			}),
+			correctAutomatically: decorateMenuItem({
```

Comment:
Where did you get this menu item from? I don't see it when misspelling a word in Safari.

Reply:
It was in the image, as on comment:
https://github.com/sindresorhus/electron-context-menu/issues/11#issuecomment-582300971